python/openchange/migration/directory.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# OpenChangeDB DB schema and its migrations
+# Copyright (C) Enrique J. Hernández Blasco <ejhernandez@zentyal.com> 2015
```

Comment:
The Copyright and the package description is incorrect. Could you copy and paste with more caution next time?